src/Components/CorrelationsReport.svelte

```diff
@@ -0,0 +1,271 @@
+<script lang="ts">
+import { PrintableCorrelation } from "src/interfaces";
+
+	import { buildReportCorrs, top3NegativeCorrs, top3PositiveCorrs } from "src/reportViewUtils";
+	import CorrelationsReportView from "../CorrelationsReportView";
+	import { menuForChartNStatsModal } from "../utils";
+
+	export let view: CorrelationsReportView;
+
+	const { plugin, app } = view;
+	const { index } = plugin;
+	const { corrs } = index;
+
+	let [lower, medium, max, min] = [0.5, 0.75, Infinity, -Infinity];
+	let [showStandards, showTopPos, showTopNeg, showStrong, showMedium, showWeak] = [true, true, true, true, true, false]
+	let corrsToShow: PrintableCorrelation[] = [];
+	let topPos3: PrintableCorrelation[] = [];
+	let topNeg3: PrintableCorrelation[] = [];
+
+	const calculateReport = () => {
+		corrsToShow = buildReportCorrs(corrs, max, min);
+		topPos3 = top3PositiveCorrs(corrsToShow);
+		topNeg3 = top3NegativeCorrs(corrsToShow);
+	}
+
+	const getButtonText = (show: boolean): string => {
+		if (show) {
+			return "↓";
+		}
+		return "↑";
+	}
+
+	const ariaShowText = (show: boolean): string => {
+		if (show) {
+			"Hide section";
+		}
+		return "Show/hide section";
```

Comment:
I haven't been able to get this to work. It's not changing the aria label when the variable changes.